src/main/java/com/example/project/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,22 @@
 package com.example.project.dto;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
It is better to use the @NotEmpty or @NotBlank for Strings
https://www.baeldung.com/java-bean-validation-not-null-empty-blank

Reply:
great advice